requirements/base.txt

```diff
@@ -50,14 +50,17 @@ pluggy==1.0.0
     # via pytest
 py==1.11.0
     # via pytest
-pyparsing==3.0.6
+pyparsing==3.0.7
     # via packaging
 pytest==6.2.5
     # via
     #   -r requirements/base.in
     #   pytest-aiohttp
-pytest-aiohttp==0.3.0
+    #   pytest-asyncio
+pytest-aiohttp==1.0.3
```

Comment:
No major change in the new version: https://github.com/aio-libs/pytest-aiohttp/blob/master/CHANGES.rst#103-2022-01-03